angular/projects/spark-angular/src/lib/components/inputs/sprk-selection-item-container/sprk-selection-item-container.component.ts

```diff
@@ -59,10 +60,26 @@ export class SprkSelectionItemContainerComponent implements OnInit {
     return classArray.join(' ');
   }
 
+  /**
+   * @ignore
+   */
+  addInputTypeClasses(): void {
+    const inputType = this.input.ref.nativeElement.type;
+    if (inputType === 'checkbox') {
+      this.renderer.addClass(this.input.ref.nativeElement, 'sprk-b-Checkbox__input');
+      this.renderer.addClass(this.label.ref.nativeElement, 'sprk-b-Checkbox__label');
+    }
+    if (inputType === 'radio') {
+      this.renderer.addClass(this.input.ref.nativeElement, 'sprk-b-Radio__input');
+      this.renderer.addClass(this.label.ref.nativeElement, 'sprk-b-Radio__label');
+    }
+  }
+
   ngOnInit(): void {
     if (this.label && this.input) {
-      this.label.ref.nativeElement.setAttribute('for', this.input_id);
-      this.input.ref.nativeElement.id = this.input_id;
+      this.renderer.setAttribute(this.label.ref.nativeElement, 'for', this.input_id);
```

Comment:
does this account for them supplying their own id?

Reply:
Will have to look into it. I did not change the functionality of the id/for. Just updated to use renderer2.